tools/visceral-evaluatesegmentation/visceral-evaluatesegmentation.xml

```diff
@@ -1,7 +1,7 @@
 <tool id="visceral_evaluatesegmentation" name="Visceral - Evaluate Segmentation" version="0.5">
   <description>Evaluate Segmentation</description>
   <requirements>
-    <requirement type="package">visceral-evaluatesegmentation</requirement>
+    <requirement type="package" version="v2015.07.02">visceral-evaluatesegmentation</requirement>
```

Comment:
Das v muss weg. Diese version gibt es sonst nicht https://anaconda.org/search?q=visceral-evaluatesegmentation

Reply:
@ThomasWollmann can you change this to version="2015.07.03"